css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
+  top: 0 !important;
```

Comment:
removed. was just testing if would stop the issue i mentioned in the previous comment. only `position` should have important now.